README.md

```diff
@@ -1 +1,76 @@
-# environment-migration-test
\ No newline at end of file
+# environment-migration-test
+
+# Environment Migration Script User Guide
+
+Required input:
+
+* Source Repo Name
+* Target Repo Name
+* Source Repo Org Name
+* Target Repo Org Name
+* Personal Access Token For Source (repo, users, and org:admin)
+* Personal Access Token For Target (repo, users, and org:admin)
+
+Inputs will be imported as environment variables with the following names:
+
+* SOURCE_REPO
+* TARGET_REPO
+* SOURCE_ORG
+* TARGET_ORG
+* GH_PAT_SOURCE
+* GH_PAT_TARGET
+
+Once inputs are entered the script will migrate the following settings:
+
+* Secrets (name only, an issue will be opened to list secrets that need to be updated)
+* Variables (ame and value)
+* Protection rules (except reviewers, an issue will be opened listing reviewers by their login and will need to be added manually)
+* Environment Name (optional repo ID tag at the end by changing a boolean in the code)
+
+Script will migrate all environments from a source org to a target org and attempt to migrate most of the settings. Certain features were not
+implemented since required reviewers may require org level permissions from a PAT (Personal Access Token) that we didn't have access to.
+
+## Running in Github Actions
+
+This option may be better since you can just save the action and script in a repo and it will run once a push is made
+
+1. Ensure all Environment variables are set in the repo level secrets
+
+The following secrets must be set in the repo:
+
+* SOURCE_REPO
+* TARGET_REPO
+* SOURCE_ORG
+* TARGET_ORG
+* GH_PAT_SOURCE
+* GH_PAT_TARGET
+
+2. Push changes to github
+
+Once changes are pushed the scirpt will run and generate the issues that will alert repo owners that secrets and reviewers need to be updated.
+When all issues have been closed you can close the PR and all environments should be migrated.
```

Comment:
Typo but also don't think we need this extra bit of info

Reply:
done